scripts/build.sh

```diff
@@ -8,4 +8,4 @@ rm -rf public/
 rm -rf content/tutorials/kit-assembly.files
 cp -r static/img/assembly content/tutorials/kit-assembly.files
 
-hugo -v
+hugo -v --gc
```

Comment:
Curiosity: what does `--gc` do?

Reply:
It removes the build cache of static files before building the site. Resizing images is fairly trivial, and this removes any issues which may come from changes in the resizing